pyproject.toml

```diff
@@ -63,8 +64,10 @@ build-backend = "setuptools.build_meta"
 write_to = "src/superphot_plus/_version.py"
 
 [tool.pytest.ini_options]
+addopts = ["--nbval-lax", "--nbval-current-env"] # Run notebooks and fail on errors, do not validate cell outputs
 testpaths = [
     "tests",
+    "docs/notebooks/*.ipynb",
```

Comment:
Doing this takes my local pytest run from 36s to 70s. The notebook tests should definitely be run in the github workflows, but I'd argue that having them run locally every time you want to run unit tests is a hindrance to development speed.

Reply:
I'm ok removing it from here and adding it to the GH actions workflows